_search-plugins/sql/full-text.md

```diff
@@ -323,13 +323,13 @@ The **^** lets you *boost* certain fields. Boosts are multipliers that weigh mat
 
 ### Syntax
 
-The syntax allows to specify the fields in double quotes, single quotes,  surrounded by backticks, or unquoted. Use star ``"*"`` to search all fields. Star symbol should be quoted.
+The syntax allows to specify the fields in double quotes, single quotes, surrounded by backticks, or unquoted. Use star ``"*"`` to search all fields. Star symbol should be quoted.
 
 ```sql
 simple_query_string([field_expression+], query_expression[, option=<option_value>]*)
 ```
 
-The weight is optional and is specified after the field name. It could be delimited by the `caret` character -- `^` or by whitespace. Please, refer to examples below:
+The weight is optional and is specified after the field name. It could be delimited by the `caret` character -- `^` or by whitespace. Refer to the following examples:
```

Comment:
```suggestion
The weight is optional and is specified after the field name. It could be delimited by the `caret` character -- `^` or by white space. Refer to the following examples:
```